app/gui/view/component-browser/component-group/src/entry.rs

```diff
@@ -0,0 +1,131 @@
+//! This module defines an entry in a component group view.
+//!
+//! The entry data is represented by the [`Model`] and visualized by the [`View`].
+
+use ensogl_core::prelude::*;
+
+use enso_frp as frp;
+use ensogl_core::application::Application;
+use ensogl_core::data::color::Rgba;
+use ensogl_core::display;
+use ensogl_core::display::scene::Layer;
+use ensogl_core::display::style;
+use ensogl_core::display::Scene;
+use ensogl_list_view as list_view;
+use ensogl_list_view::entry::Label;
+
+
+
+// ===============
+// === Aliases ===
+// ===============
+
+/// ID of a component group entry in a ListView.
+pub type Id = list_view::entry::Id;
+
+
+
+// =============
+// === Model ===
+// =============
+
+/// Data underlying an entry in a component group view.
+#[allow(missing_docs)]
+#[derive(Clone, Debug, Default)]
+pub struct Model {
+    pub label: String,
+}
+
+impl From<String> for Model {
+    fn from(label: String) -> Self {
+        Model { label }
+    }
+}
+
+impl From<&str> for Model {
+    fn from(label: &str) -> Self {
+        label.to_owned().into()
+    }
+}
+
+
+
+// ==============
+// === Params ===
+// ==============
+
+/// A type parametrizing the visual aspects of how an entry will be rendered in an instance of
+/// component group view.
+#[allow(missing_docs)]
+#[derive(Clone, CloneRef, Debug)]
+pub struct Params {
+    pub color: frp::Sampler<Rgba>,
+}
+
+impl Default for Params {
+    fn default() -> Self {
+        let network = frp::Network::new("component_group::Params::default");
+        frp::extend! { network
+            color_source <- source::<Rgba>();
+            color <- color_source.sampler();
+        }
+        Self { color }
+    }
+}
+
+
+
+// ============
+// === View ===
+// ============
+
+/// A visual representation of a [`Model`].
+#[derive(Clone, CloneRef, Debug)]
+pub struct View {
+    logger:         Logger,
+    display_object: display::object::Instance,
+    label:          Label,
+}
+
+impl list_view::Entry for View {
+    type Model = Model;
+    type Params = Params;
+
+    fn new(app: &Application, style_prefix: &style::Path, params: &Self::Params) -> Self {
+        let logger = Logger::new("component-group::Entry");
+        let display_object = display::object::Instance::new(&logger);
+        let label = Label::new(app, style_prefix, &());
+        display_object.add_child(&label);
+
+        let network = &label.network;
+        let color = params.color.clone();
+        let label_frp = &label.label.frp;
+        frp::extend! { network
+            init <- source_();
+            color <- all(&color, &init)._0();
+            label_frp.set_color_all <+ color;
```

Comment:
I think this line is not needed.

Reply:
Based on our discussion, I'm leaving this as-is for now. There's some refactoring needed in `text::Area` before changing this, and the refactoring is being postponed for a later time (to be tracked by @farmaazon).

Reply:
Then this line should have a comment about that. If this should not be needed but is needed right now, a fixme with a description should be provided in the code. How otherwise e will remember about it?